containerbuddy/config.go

```diff
@@ -311,8 +312,10 @@ func initializeConfig(config *Config) (*Config, error) {
 			return nil, ifaceErr
 		}
 
-		if service.ipAddress, err = GetIP(interfaces); err != nil {
-			return nil, err
+		if service.Ip == "" {
```

Comment:
We're setting the new `Ip` field on the `ServiceConfig` struct but then that field isn't being consumed anywhere. So this will let us get the raw value from the config file but then we're not doing anything with it.